### ISSUE TITLE
storybook for the front-end 

### ISSUE THREAD

**Issue by polaroidkidd** [LibrePhotos/librephotos#635]
**Describe the enhancement you'd like**
Add storybook the the development process.

**Describe why this will benefit the LibrePhotos**
Changing inputs and behaviour of components can be done in an isolated environment while remaining sure that all consumers adhere to the components' api

**Additional context**
N/A

**Comment by sickelap** [comment]
I like the idea to have a storybook. @polaroidkidd are you planning to pick this up anytime soon?

**Comment by polaroidkidd** [comment]
> anytime soon?

Might start giving it a go this weekend. I can start with a couple of components to serve as a template (or a "how-not-to-do-it" reference since I've never worked with storybook) and then others can contribute to the book. If we want to actively maintain it, then new components should always include a story-book, right?

**Comment by sickelap** [comment]
Yes, new components should contain a storybook so we have a clear example of how to use it.
For me, it's difficult to see us adding a lot of new components right now. I guess at the beginning we would be refactoring existing ones.

**Comment by polaroidkidd** [comment]
should we maybe add refactored components to the book too as part of the refactoring?

**Comment by sickelap** [comment]
sure. Just pick a component that is definitely used in multiple places, like [Tile](https://github.com/LibrePhotos/librephotos-frontend/blob/dev/src/components/Tile.tsx)
